kratos/tests/cpp_tests/geometries/test_line_2d_3.cpp

```diff
@@ -22,60 +22,60 @@
 #include "tests/cpp_tests/geometries/test_shape_function_derivatives.h"
 #include "tests/cpp_tests/geometries/cross_check_shape_functions_values.h"
 
-namespace Kratos {
-namespace Testing {
-
-    /// Factory functions
-
-    /** Generates a point type sample Line2D3N
-    * @return  Pointer to a Line2D3N
-    */
-    Line2D3<Point>::Pointer GeneratePointsUnitXDirectionLine2D3() {
-        return Kratos::make_shared<Line2D3<Point>>(
-        Kratos::make_shared<Point>(0.0, 0.0, 0.0),
-        Kratos::make_shared<Point>(1.0, 0.0, 0.0),
-        Kratos::make_shared<Point>(0.5, 0.0, 0.0)
-        );
-    }
-
-    /** Generates a point type sample Line2D3N
-    * @return  Pointer to a Line2D3N
-    */
-    Line2D3<Point>::Pointer GeneratePointsUnitYDirectionLine2D3() {
-        return Kratos::make_shared<Line2D3<Point>>(
-        Kratos::make_shared<Point>(0.0, 0.0, 0.0),
-        Kratos::make_shared<Point>(0.0, 1.0, 0.0),
-        Kratos::make_shared<Point>(0.0, 0.5, 0.0)
-        );
-    }
+namespace Kratos::Testing {
+    namespace {
```

Comment:
Just for the record, at some point we agreed to avoid indenting the namespaces for the sake of readability.

Reply:
Oki

Reply:
Done